package/gluon-dns-cache/check_site.lua

```diff
@@ -0,0 +1,3 @@
+need_string_match('dns.cacheentries', '^%d$')
+need_string_array('dns.servers')
```

Comment:
I don't like hardcoding the DNS servers, but I guess there aren't really better options that don't requite much more work...